ota-plus-web/app/reactapp-new/src/networking/UserNetworking.ts

```diff
@@ -8,7 +8,7 @@ export interface UserNetworkingInterface {
 export class UserNetworking implements UserNetworkingInterface {
   public async getUserProfile(): Promise<UserProfile> {
     // FIXME: until we do not have axios and networking configurated the url is hardcoded
```

Comment:
The comment should be removed as well I presume?

Reply:
No, this is FIXME. Application is under development stage and there are many features which are not implemented yet (like http connections like in this file). Wnen network connector will be developed this will be removed.